### PR TITLE
Add `loading` attribute to the `Image` component

### PR DESCRIPTION
**Pull request by cavaglieridomenico**
#### What problem is this solving?

This provides the ability to lazy load the individual Image component. The loading attribute value is a prop to pass from Theme.

The goal is to achieve an improvement in performance.

#### How to test it?

[Workspace]( https://seopod908--itwhirlpool.myvtex.com/asciugatrice-a-pompa-di-calore-whirlpool-a-libera-installazione-9-0-kg-st-u-92e-eu-869991583690/p)

#### Screenshots or example usage:

wpit-lazy-loading
wpit-lazy-loading-theme

### CODE REVIEW

**Comment by cavaglieridomenico** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by cavaglieridomenico** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [ ]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by cavaglieridomenico** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [ ]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉



> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by cavaglieridomenico** [comment]
> Hi! I'm VTEX IO CI/CD Bot and I'll be helping you to publish your app! 🤖
> 
> Please select which version do you want to release:
> 
> * [ ]  Patch (backwards-compatible bug fixes)
> * [x]  Minor (backwards-compatible functionality)
> * [ ]  Major (incompatible API changes)
> 
> And then you just need to merge your PR when you are ready! There is **no need** to create a release commit/tag.
> 
> * [ ]  No thanks, I would rather do it manually 😞

**Comment by cavaglieridomenico** [comment]
> Beep boop 🤖
> 
> I noticed you didn't make any changes at the `docs/` folder
> 
> * [x]  There's nothing new to document 🤔
> * [ ]  I'll do it later 😞
> 
> In order to keep track, I'll create an issue if you decide now is not a good time
> 
> * [ ]  I just updated 🎉🎉

**Comment by cavaglieridomenico** [comment]
Hi, I made some changes, is it possible to review them?

Thank you.

File: store/contentSchemas.json
Comment (filipewl) [inline store/contentSchemas.json:1]: Added the possibility of editing the `loading` prop via Site Editor.
SiteEditor

**Review by filipewl — APPROVED** [review]
Looks good to me! Sorry for the delay.

I've added the possibility of also setting the new `loading` prop via Site Editor.

File: store/contentSchemas.json
Comment (cavaglieridomenico) [reply store/contentSchemas.json:1]: Good idea! Thank you!

**Pull request merged**